src/discounts/components/DiscountCreatePage/const.ts

```diff
@@ -0,0 +1,30 @@
+import { Condition, DiscoutFormData, Rule } from "@dashboard/discounts/types";
```

Comment:
issue: is it really "const" a good for filename? perhaps name accordingly to what it holds